terraform/deployments/govuk-publishing-platform/frontends_origins.tf

```diff
@@ -136,6 +145,13 @@ resource "aws_lb_target_group" "draft_router" {
   health_check {
     path = "/"
   }
+
+  tags = merge(
+    local.additional_tags,
+    {
+      name      = "frontend-origin-draft-router-tg"
```

Comment:
based on my new #231, it should be just `draft-router-tg`

Reply:
ok